### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by saketkum11**
Please Review
- carts 
image

### CODE REVIEW

File: component/cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    <script src="https://kit.fontawesome.com/85de537a36.js" crossorigin="anonymous"></script>
+    <title>Cart</title>
+</head>
+
+<body>
+    <div class="h-100">
+
+
+        <!--- Navigation bar -->
+        <header class=" bg-green-7 text-color-0 pd-5 position-fixed t-0">
```
Comment (znjs) [inline component/cart/cart.html]: if possible use semantic HTML here.
wrap the whole nav bar in `nav` tag

File: component/cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                        class="pd-y-3 pd-x-5 rounded-L border-none text-center outline-none">
+                    <i class="fa-solid fa-magnifying-glass bg-black-0 text-color-9 rounded-L pd-3 text-xm"></i>
+                </div>
+                <div class="m-lf">
+
+
+                    <ul class="style-none flex items-center">
+                        <li class="pd-x-4">
+                            <a href="/component/login/login.html" target="_blank"
+                                class="text-dec text-color-0 pd-x-5 pd-y-3 bg-red-7">Login</a>
```
Comment (znjs) [inline component/cart/cart.html]: Generally cart is visible only after logging in, if possible you can remove the login button

**Review by znjs — APPROVED** [review]
suggestion: if possible you can remove `target="_blank"` in some places user doesn't expect to open in a new window every time

File: component/cart/cart.html (hunk 2 above)
Comment (saketkum11) [reply component/cart/cart.html]: Ya sure Srikar.

File: component/cart/cart.html (hunk 1 above)
Comment (saketkum11) [reply component/cart/cart.html]: Yes I have actually forgot to add semantic tag.
Thank you for pointing out.

**Review by shubhambishnoi — APPROVED** [review]

**Pull request merged**